incubator-tools/overlapping_split/overlapping_split.ipynb

```diff
@@ -0,0 +1,473 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    def get_coordinate(entity: documentai.Document.Entity, arg: str) -&gt; int:
</pre><p>Move to utilities.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>